hosted-ce/Dockerfile

```
@@ -0,0 +1,45 @@
+#############
+# hosted-ce #
+#############
+
+FROM base:latest AS hosted-ce
```

```suggestion
FROM base:latest
```

Same comment as before about removing `AS ...`. Also, the cool thing about GitHub suggestions is that you can commit the suggestion or batch multiple suggestions into a single commit! One perk of that is that the suggester gets co-author on the commit :)